sections/sections-app/src/bundle/sections.properties

```diff
@@ -35,6 +35,7 @@ edit_student_successful=Students in {0} were updated successfully\!
 section_max_size=Section Size
 student_view_join=Join
 edit_student_unassigned=Unassigned Students
+students_not_found=No related students found in this site.
```

Comment:
Since this is a search function, maybe we should say something like "No matching students found..." rather than "related"?

CC: @fostersdesign

Reply:
I see what you mean. Perhaps "No matching students found..." for when the search returns nothing and "There are currently no students enrolled in this site" for when there are no members?

@fostersdesign, thoughts?

Reply:
Ya, those options sound good, @bjones86. Thanks.

Reply:
Hey, I can't get the second message to be displayed in case there are no students on the Site.
If it is not enough to show the two messages together in both situations, maybe I should leave this to someone else.